Core.ApplicationServices/GDPR/IDataProcessingRegistrationApplicationService.cs

```diff
@@ -21,6 +21,7 @@ public interface IDataProcessingRegistrationApplicationService
         Result<IQueryable<DataProcessingRegistration>, OperationError> GetOrganizationData(int organizationId, int skip, int take);
         Result<DataProcessingRegistration, OperationError> UpdateName(int id, string name);
         Result<ExternalReference, OperationError> SetMasterReference(int id, int referenceId);
+        Result<ExternalReference, OperationError> ClearMasterReference(int id);
```

Comment:
Det er en skidt kontrakt for den kan jo godt være null. Jeg ville nok vælge `Maybe<OperationError>` i stedet for på interface niveau. eller `Result<Maybe<ExternalReference>, OperationError>` for at angive at det kun er måske at referencen var der